benchmark/src/main/scala/code/winitzki/benchmark/Benchmarks9.scala

```diff
@@ -107,36 +107,81 @@ object Benchmarks9 {
     result
   }
 
+  val counterMultiplier = 10
+
```

Comment:
The benchmarks should be largely unaffected by the overhead of creating a ThreadPool. They are completely dominated by other things. I'll see if `tp` is inconsistent across benchmarks.